Components/alert/alert.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>alert - minimal ui</title>
+    <link rel="stylesheet" href="/Components/install/install.css" />
+    <link rel="stylesheet" href="/prism.css" />
+    <link rel="stylesheet" href="/minimal.css" />
+    <link rel="stylesheet" href="alert.css" />
```

Comment:
You can import all CSS files in a single CSS file.

Reply:
Thanks
Implemented it